### PR TITLE
Move "compose" on mobile to floating action button

### PR DESCRIPTION
**Pull request by Gargron**
screen shot 2018-03-02 at 05 52 31

### CODE REVIEW

**Review by akihikodaki — APPROVED** [review]
I am a bit surprised seeing you make this change after versioning RC.

**Comment by Gargron** [comment]
@akihikodaki Until a git tag exists, the version is not immutable/finalized. Sometimes I bump it in anticipation...

File: app/javascript/styles/mastodon/components.scss [hunk 1]
```diff
@@ -5017,3 +5016,27 @@ noscript {
...
+
+.floating-action-button {
+  position: fixed;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 3.9375rem;
+  height: 3.9375rem;
+  bottom: 1.3125rem;
+  right: 1.3125rem;
```
Comment (unarist) [inline app/javascript/styles/mastodon/components.scss]: @Gargron Why did you use rem which we haven't used before?

**Comment by zcdunn** [comment]
So where would the search field be with this change?

**Comment by Gargron** [comment]
@zcdunn Search is still on the compose page, nothing changed about that.

File: app/javascript/styles/mastodon/components.scss (hunk 1 above)
Comment (Gargron) [reply app/javascript/styles/mastodon/components.scss]: I looked up how the floating action button was positioned in another Material UI application :thinking: I figured the button needs to be easily clickable so I shouldn't mess with these values.

**Comment by zcdunn** [comment]
So you can't swipe between home and compose columns now? It seems pretty weird to have to hit a button with a pencil on it to search.

**Pull request merged**